src/assignments/5-loop.js

```diff
@@ -20,6 +30,11 @@ sum=8
 
 */
 // start here writing the code for program2
+ for(let number = 0; number <= 125; number++){
+    if(number%2 == 0){
+        console.log(number)
+    }
+ }
```

Comment:
https://stackoverflow.com/questions/38334652/sum-all-the-digits-of-a-number-javascript
You can the the logic solution form here, and redo the task again!